src/datasources/work-orders/constants/WorkOrdersQueryBuilder.constants.ts

```diff
@@ -0,0 +1,170 @@
+    UpdatedAt = 'updatedAt',
+    AssignedTo = 'assignedTo',
+    RequestedBy = 'requestedBy',
+    CreatedBy = 'createdBy',
+    UpdatedBy = 'updatedBy',
+  }
+
+export const WorkOrdersQueryBuilderFields: Record<string, QBField> = {
+    NAME: {
+        label: 'Name',
+        dataField: WorkOrdersQueryBuilderFieldNames.Name,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name,
+            QueryBuilderOperations.CONTAINS.name,
+            QueryBuilderOperations.DOES_NOT_CONTAIN.name
+        ]
+    },
+    DESCRIPTION: {
+        label: 'Description',
+        dataField: WorkOrdersQueryBuilderFieldNames.Description,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name
+        ]
+    },
+    WORK_ORDER_ID: {
+        label: 'Work Order ID',
+        dataField: WorkOrdersQueryBuilderFieldNames.WorkOrderID,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name
+        ]
+    },
+    STATE: {
+        label: 'State',
+        dataField: WorkOrdersQueryBuilderFieldNames.State,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name
+        ]
+    },
+    TYPE: {
+        label: 'Type',
+        dataField: WorkOrdersQueryBuilderFieldNames.Type,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name
+        ]
+    },
+    WORKSPACE: {
+        label: 'Workspace',
+        dataField: WorkOrdersQueryBuilderFieldNames.Workspace,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name
+        ]
+    },
+    EARLIEST_START_DATE: {
+        label: 'Earliest Start Date',
+        dataField: WorkOrdersQueryBuilderFieldNames.EarliestStartDate,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name,
+            QueryBuilderOperations.GREATER_THAN.name,
+            QueryBuilderOperations.GREATER_THAN_OR_EQUAL_TO.name,
+            QueryBuilderOperations.LESS_THAN.name,
+            QueryBuilderOperations.LESS_THAN_OR_EQUAL_TO.name,
+        ]
+    },
+    DUE_DATE: {
+        label: 'Due Date',
+        dataField: WorkOrdersQueryBuilderFieldNames.DueDate,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name,
+            QueryBuilderOperations.GREATER_THAN.name,
+            QueryBuilderOperations.GREATER_THAN_OR_EQUAL_TO.name,
+            QueryBuilderOperations.LESS_THAN.name,
+            QueryBuilderOperations.LESS_THAN_OR_EQUAL_TO.name,
+        ]
+    },
+    CREATED_AT: {
+        label: 'Created At',
```

Comment:
```suggestion
        label: 'Created',
```

Can we match it to the query builder in the work orders grid?

![image](https://github.com/user-attachments/assets/a673eb18-1ecf-4cf6-a8eb-e480b11ce922)